src/main/webapp/app/localci/build-agents/build-agents.component.html

```diff
@@ -76,3 +76,139 @@ <h3 id="build-agents-heading" jhiTranslate="artemisApp.buildAgents.title">Build
+                            </span>
+                        </ng-template>
+                        <ng-template ngx-datatable-cell-template let-value="value">
+                            <span>{{ value }}</span>
+                        </ng-template>
+                    </ngx-datatable-column>
+                    <ngx-datatable-column prop="priority" [minWidth]="150" [width]="200">
+                        <ng-template ngx-datatable-header-template>
+                            <span class="datatable-header-cell-wrapper" (click)="controls.onSort('priority')">
+                                <span class="datatable-header-cell-label bold sortable" jhiTranslate="artemisApp.buildQueue.buildJob.priority"> Priority </span>
+                                <fa-icon [icon]="controls.iconForSortPropField('priority')"></fa-icon>
+                            </span>
+                        </ng-template>
+                        <ng-template ngx-datatable-cell-template let-value="value">
+                            <span>{{ value }}</span>
+                        </ng-template>
+                    </ngx-datatable-column>
+                    <ngx-datatable-column prop="buildCompletionDate" [minWidth]="150" [width]="200">
+                        <ng-template ngx-datatable-header-template>
+                            <span class="datatable-header-cell-wrapper" (click)="controls.onSort('buildCompletionDate')">
+                                <span class="datatable-header-cell-label bold sortable" jhiTranslate="artemisApp.buildQueue.buildJob.buildCompletionDate">
+                                    Build Completion Date
+                                </span>
+                                <fa-icon [icon]="controls.iconForSortPropField('buildCompletionDate')"></fa-icon>
+                            </span>
+                        </ng-template>
+                        <ng-template ngx-datatable-cell-template let-value="value">
+                            <span>{{ value | artemisDate: 'long' : true }}</span>
+                        </ng-template>
+                    </ngx-datatable-column>
+                    <ngx-datatable-column prop="result" [minWidth]="150" [width]="200">
+                        <ng-template ngx-datatable-header-template>
+                            <span class="datatable-header-cell-wrapper" (click)="controls.onSort('result')">
+                                <span class="datatable-header-cell-label bold sortable" jhiTranslate="artemisApp.buildQueue.buildJob.result"> Result </span>
+                                <fa-icon [icon]="controls.iconForSortPropField('result')"></fa-icon>
+                            </span>
+                        </ng-template>
+                        <ng-template ngx-datatable-cell-template let-value="value" let-row="row">
+                            <span [ngStyle]="{ color: getBuildJobNameColor(row.result) }">{{ value }}</span>
```

Comment:
I think it would be better to use the bootstrap classes text-danger and text-success here instread of directly specifying the colors. With a `ngClass` you could then also get rid of the function entirely:
```suggestion
                            <span [ngClass]="row.result === 'SUCCESSFUL' ? 'text-success' : 'text-danger'">{{ value }}</span>
```